build.sbt

```diff
@@ -115,6 +115,68 @@ lazy val server = project
   .settings(moduleName := "mu-rpc-server")
   .settings(serverSettings)
 
+/////////////////////
+//// HEALTHCHECK ////
+/////////////////////
+
+lazy val `health-check-unary` = project
+  .in(file("modules/health-check/health-check-unary"))
+  .dependsOn(channel)
+  .dependsOn(server)
+  .settings(healthCheckSettings)
+  .settings(moduleName := "mu-rpc-health-check-unary")
+
+lazy val `health-check-monix` = project
+  .in(file("modules/health-check/health-check-monix"))
+  .dependsOn(`health-check-unary`)
+  .dependsOn(monix)
+  .settings(healthCheckSettingsMonix)
+  .settings(moduleName := "mu-rpc-health-check-monix")
+
+lazy val `health-check-fs2` = project
```

Comment:
I'd add the code to the `internal-core`, `internal-monix`, and `internal-fs2`, modules respectively

Reply:
Sure, what are they providing? I'm not quite sure...

Reply:
umm, maybe `channel`, `monix`, and `fs2` are better options

Reply:
@mrtmmr this is still pending, I prefer to avoid extra modules, especially if the new feature (health-check) doesn't need a new dependency

Reply:
Do you mean `server` and `channel` for `health-check-unary`? I made some tests and I need both

Reply:
I mean to add the unary code to the channel module

Reply:
Indeed I think we could put the three examples in one single module